distribution/ddf-common/src/main/resources/etc/transformers/ddf-default-input-transformers.json

```diff
@@ -0,0 +1,16 @@
+{
+  "transformerIds": [
```

Comment:
I preferred no documentation because I don't think admins should ever have any concern about this, only developers. I could be convinced otherwise though.

Reply:
@ricklarsen opinion?

Reply:
❓ Why not manually create the set of strings in InputTransformerIds so we don't need to read from this file?